tests/phpunit/payment-methods/class-wc-stripe-express-checkout-ajax-handler-test.php

```
@@ -197,6 +197,102 @@ public function test_ajax_add_to_cart_returns_success_for_simple_product() {
+		} finally {
+			Ajax_Test_Helper::remove_hooks();
+			unset( $_POST['product_id'], $_POST['qty'], $_POST['addon_value'], $_POST['security'], $_REQUEST['security'] );
+		}
+
+		// $10 product + $5 add-on = $15 tax base; with the stubbed $1.50 tax the total is $16.50.
+		$this->assertIsArray( $response );
+		$this->assertArrayHasKey( 'displayItems', $response, 'response: ' . wp_json_encode( $response ) );
+		$this->assertSame( 1500, $response['displayItems'][0]['amount'] );
+		$this->assertSame( 150, $response['displayItems'][1]['amount'] );
+		$this->assertSame( 1650, $response['total']['amount'] );
+	}
```

PHPMD (2.15.0)

[error] test_ajax_get_selected_product_data_includes_addons_in_tax_base accesses the super-global variable $_REQUEST. (undefined)

(Superglobals)